### ISSUE TITLE
Move repository out of private GitHub account

### ISSUE THREAD

**Issue by juliusv** [juliusv/webhook-logger#8]
We should probably move this repository to a GitHub organization owned by Red Hat, as that is who I expect will own it in the long term. @simon3z Can you provide guidance on this? Then we can follow https://help.github.com/articles/about-repository-transfers/ to move it.

**Comment by simon3z** [comment]
@juliusv do you think this could be transferred under the Prometheus organization here in GitHub?
Of course the amount of Red Hat involvement in maintaining this won't change.

**Comment by juliusv** [comment]
@simon3z I'd prefer for this to not live in the Prometheus organisation for two reasons:

- One one hand the tool is very generic now (no direct relationship to Prometheus), while on the other hand, the AM-related use case is pretty specific again, and it's unclear whether many others will need that specific webhook message buffer integration.

- We have to provide some base level of maintenance for any repositoy that's in our org (common coding conventions, build infrastructure, etc.).

So I think it would make more sense for it to live in a Red-Hat-owned org.

**Comment by simon3z** [comment]
@juliusv OK give me few days to orchestrate the new repository.

**Comment by simon3z** [comment]
@juliusv the current idea is to transfer this to `openshift/prometheus-alert-buffer`. IIRC the transfer request should come from you, then I don't know if there is something the openshift org has to do to accept it.
Anyway cc @smarterclayton

**Comment by juliusv** [comment]
@simon3z Ok, I can rename it to `prometheus-alert-buffer` here first and then initiate the transfer.

Is the name final? One thing to think about: do you want to keep this tool generic or not? If it is supposed to be a generic tool, naming it `prometheus-alert-buffer` will make it look like it can only be used for buffering Prometheus alerts and nobody will ever think of using it for anything else?

**Comment by simon3z** [comment]
> Is the name final? One thing to think about: do you want to keep this tool generic or not? If it is supposed to be a generic tool

That was my plan as well :+1: 

> naming it `prometheus-alert-buffer` will make it look like it can only be used for buffering Prometheus alerts and nobody will ever think of using it for anything else?

This is understood. Not sure if @smarterclayton wants to discuss this again.

(I guess that if by mid next week this thread is not revived we should probably take it as final)

**Comment by juliusv** [comment]
Sounds good. I'll just let this sit here for now then and do the rename after the final decision, then request the move.

**Comment by zgalor** [comment]
@smarterclayton, Are you OK with the name `prometheus-alert-buffer`?
Can you move this repository under https://github.com/openshift/ ?

**Comment by simon3z** [comment]
@zgalor @smarterclayton @juliusv I think we can go with prometheus-alert-buffer. Later (much later :blush:) we can then extract the core (as we speak 100% :-) to a generic go lib/service called `message-buffer` if we really want to.

Anyway let's focus on the priorities and move this to `openshift/prometheus-alert-buffer`.

**Comment by juliusv** [comment]
@simon3z I tried transfer the repo to openshift, but I got an error:

"You don’t have the permission to create repositories on openshift"

I don't know, maybe you need to initiate the move as someone who has rights in the "openshift" org. I invited you (@simon3z and @ilackarms) to be collaborators on this repo here so I can then give you admin access to move it (hopefully). Please accept the invites.

**Comment by simon3z** [comment]
@juliusv I accepted the invite but I don't see the repository settings page (too few permissions?).

Anyway I think you should invite @smarterclayton who may be able to create new repos there.

**Comment by juliusv** [comment]
@simon3z Indeed. I was hoping that once you accepted the invite, I could give you more rights, but I don't see such an option in the collaborator interface.

Unless you want to give me access to the "openshift" org, I would recommend for one of you with access to it (@smarterclayton?) to just clone the git repo and push it to its new destination manually. It means you lose old pull requests and issues, but maybe that's ok.

**Comment by juliusv** [comment]
@simon3z @smarterclayton Is there any news here about the new repo home? See my comment above.
